### DISCUSSION TITLE
Building GCC on OS X

### DISCUSSION THREAD

**Discussion by amneetb** [IBAMR/IBAMR#1427] [Q&A]
@boyceg @knepley @drwells: I am trying to build gcc on OS X with openmp. Do I need to include special configure flags for it? Unfortunately, Apple Clang does not come with libomp.

**Comment by amneetb** [comment]
(I did not see anything related to openmp with `./configure --help`)

**Reply by knepley** [reply]
I do not remember doing anything special. However, on OSX, gdb does not work and there are other problems with the DWARF format. It makes gcc a lot less usable. It is almost just better to run a Linux container.

**Comment by drwells** [comment]
I don't know. I'd take a look at how spack compiles gcc to get some hints.

**Reply by amneetb** [reply]
I was able to build it successfully. There was no need to explicitly tell gcc about libomp.
